src/pages/our-organisation/index.tsx

```diff
@@ -218,11 +221,11 @@ const HomePage = () => (
       <>
         <InnerText title="Our people" theme="light">
           <p>
-            The majority of our staff, around 90% of our workforce, are employed
-            in the UK, and we have growing operations in the US and Australia.
-            We value and respect all differences in all people, seen and unseen,
-            and aspire to an open, supportive and inclusive culture which makes
-            the Guardian a good place to work for everyone.
+            We value and respect all differences in people, seen and unseen, and
+            aspire to an open, supportive and inclusive culture which makes the
+            Guardian a good place to work for everyone. We have a global
+            workforce, with THE MAJORITY OF our staff in the UK and growing
```

Comment:
```suggestion
            workforce, with the majority of our staff in the UK and growing
```

Reply:
Think they've gone with caps here for emphasis